test/unit_tests/graphql/test_process.py

```diff
@@ -39,4 +44,28 @@ def test_process(test_client, mocked_processes):
 
     response = test_client.post("/api/graphql", content=test_query, headers={"Content-Type": "application/json"})
     assert response.status_code == HTTPStatus.OK
-    assert response.json() == {"data": {"process": {"processId": str(process_id)}}}
+    assert response.json() == {
+        "data": {
+            "process": {"processId": str(process_id), "userPermissions": {"resumeAllowed": True, "retryAllowed": True}}
+        }
+    }
+
+
+@mock.patch("orchestrator.graphql.schemas.process.get_workflow")
+def test_process_is_allowed_with_historic_workflow_only_left_in_db(
+    mock_get_workflow, test_client, mocked_processes, test_workflow_soft_deleted
+):
+    mock_get_workflow.return_value = None
+    process_id = mocked_processes[0]
+    test_query = build_simple_query(process_id)
+
+    response = test_client.post("/api/graphql", content=test_query, headers={"Content-Type": "application/json"})
+    assert response.status_code == HTTPStatus.OK
+    assert response.json() == {
```

Comment:
Why not only test the relevant part of the response? I guess processId is not that interesting, just the userPermissions